### ISSUE TITLE
Missing comma in example footnote

### ISSUE THREAD

**Issue by CuriousMagpie** [KittyGiraudel/eleventy-plugin-footnotes#53]
Hey there! I just installed your plug-in and discovered that your example code is missing a comma between the ID and the text of the footnote. It should read:

```
Something about {% footnoteref "css-counters", "CSS counters are, in essence, variables maintained by CSS whose values may be incremented by CSS rules to track how many times they’re used." %}CSS counters{% endfootnoteref %} that deserves a footnote explaining what they are.
```

**Comment by KittyGiraudel** [comment]
Hello. Thank you for noticing. It’s been addressed with 19e40c394c6fde44e21a01d42fa0a427a9563773. Cheers! ✨

**Comment by CuriousMagpie** [comment]
Great! Thanks so much--I love using footnotes and was really flummoxed by
why 11ty doesn't come with that functionality outside of the box, you know?
And semantic footnotes? Even better! 💯
--
My pronouns are she/they.

I am turning in revolution
These are the scars that silence carved on me
     -- Vienna Teng, "Gravity"
